docs/platform/reference/eol-for-major-versions.rst

```diff
@@ -0,0 +1,167 @@
+EOL for major versions of Aiven Services
+========================================
+
+End of Life (EOL) refers to the deadline after which affected Aiven
+services are retired and will be no longer supported nor maintained.
+
+Since August 2020, Aiven aims to follow the EOL schedule set by the
+original authors and maintainers of the open source software, aka
+upstream projects. Once the upstream project retires a specific version,
+they do not receive security updates and critical bug fixes anymore by
+the maintainers.
+
+Continued use of outdated services means that they no longer offer our
+customers the level of protection their business needs. Therefore, by
+following the upstream project's EOL schedule, we ensure that Aiven
+services are always running on supported versions of the open source
+software.
+
+**Version numbering**
+~~~~~~~~~~~~~~~~~~~~~
+
+Aiven services inherit the upstream project’s software versioning
+scheme. Depending on the service, a major version can be either a single
+digit (e.g. PostgreSQL® 12) or ``major.minor`` (e.g. Kafka® 2.4). The
```

Comment:
Can we use more updated version of the tools? We're offering PG 14 and Kafka 3.0 at this point